### DISCUSSION TITLE
Toast service timeout not working as expected

### DISCUSSION THREAD

**Discussion by tvpkn-9192** [Blazored/Toast#204] [Q&A] [answered]
Hi Team,

Based on documentation I am trying to set TestSettings for Timeout in my service. It doesn't work for me. I am using version 4.0 currently. Please correct me if any latest version has to be applied for this issue.

Thanks!

**Comment by chrissainty** [comment, marked as answer]
This is a bug and is tracked by https://github.com/Blazored/Toast/issues/195

**Reply by tvpkn-9192** [reply]
@chrissainty  thanks! I see it's resolved. May I please know if I can expect the patch in v4.0?
